org.eclipse.jdt.debug.ui/ui/org/eclipse/jdt/internal/debug/ui/breakpoints/StandardJavaBreakpointEditor.java

```diff
@@ -125,12 +127,15 @@ public void modifyText(ModifyEvent e) {
 				setDirty(PROP_HIT_COUNT);
 			}
 		});
+
 		SWTFactory.createLabel(composite, "", 1); // spacer //$NON-NLS-1$
-		Composite radios = SWTFactory.createComposite(composite, composite.getFont(), 2, 1, GridData.FILL_HORIZONTAL, 0, 0);
+		Composite radios = SWTFactory.createComposite(composite, composite.getFont(), 3, 1, GridData.FILL_HORIZONTAL, 0, 0);
```

Comment:
I believe "Resume" shouldn't be in the "Suspend" group below. Although it is understandible why you've added it here, unlike other radio buttons, it belongs solely to the "Trigger Point" checkbox, so I would propose something like this:

![image](https://github.com/user-attachments/assets/85b160e6-4303-44e2-89fe-5dd2d461bc83)